src/languages/en.ts

```diff
@@ -557,19 +557,19 @@ export default {
         splitAmount: ({amount}: SplitAmountParams) => `split ${amount}`,
         didSplitAmount: ({formattedAmount, comment}: DidSplitAmountMessageParams) => `split ${formattedAmount}${comment ? ` for ${comment}` : ''}`,
         amountEach: ({amount}: AmountEachParams) => `${amount} each`,
-        payerOwesAmount: ({payer, amount}: PayerOwesAmountParams) => `${payer} owes ${amount}`,
+        payerOwesAmount: ({payer, amount}: PayerOwesAmountParams) => `${payer ? `${payer} ` : ''}owes ${amount}`,
```

Comment:
```suggestion
        payerOwesAmount: ({payer, amount}: PayerOwesAmountParams) => `${payer ?? ''}owes ${amount}`,
```

Reply:
but here I'm adding an additional space after payer (I don't want it when it's no payer)

Reply:
right 😅  hard to see this space, so I fine with that what you added 😄